src/components/BookAdd.jsx

```diff
@@ -18,16 +18,16 @@ function BookAdd() {
     setAuthor(e.target.value);
   };
 
-  const handlebookChange = (e) => {
-    setBook(e.target.value);
+  const handleCategoryChange = (e) => {
+    setCategory(e.target.value);
   };
 
   const submitBookToStore = () => {
     const newBook = {
-      id: uuidv4(),
+      item_id: uuidv4(),
       title,
       author,
-      type,
+      category,
     };
     dispatch(addBook(newBook));
   };
```

Comment:
[Optional] I suggest that you clear the form when an entry is submitted to allow making a new entry.

![image](https://user-images.githubusercontent.com/20554583/207840950-4781446e-f138-4fde-8f9b-4ba6c3b6367f.png)